gen/PhotonPoseEstimator.yml

```diff
@@ -0,0 +1,27 @@
+---
+
+enums:
+  PoseStrategy:
+classes:
+  EstimatedRobotPose:
+    attributes:
+      estimatedPose:
+      timestamp:
+    methods:
+      EstimatedRobotPose:
+        param_override:
+          pose_:
+            name: estimatedPose
+          time_:
+            name: timestamp
+  PhotonPoseEstimator:
+    methods:
+      PhotonPoseEstimator:
```

Comment:
This constructor takes an rvalue reference (`PhotonCamera&&`), and intentionally takes ownership, moving it.

I haven't had a chance to try to make this usable via pybind11. To unblock this release I'll disable this class for now.